android/java/org/chromium/chrome/browser/upgrade/NotificationIntent.java

```diff
@@ -0,0 +1,93 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.upgrade;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.net.Uri;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+import android.view.View;
+
+import org.chromium.base.Log;
+import org.chromium.base.ThreadUtils;
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.preferences.BravePreferenceKeys;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Fires notification about new features.
+ */
+public class NotificationIntent {
+    private static final String TAG = "NotificationIntent";
+    private static final String NOTIFICATION_TAG = "16c570a4-da7d-4c4e-8518-d2b7d6e41615";
+    private static final String NOTIFICATION_CHANNEL_ID = "a79c3102-4183-4001-a553-ec3041bd0f49";
+    private static final String URL = "https://support.brave.com/hc/en-us/articles/360045401211/";
+    private static final List<String> mWhitelistedRegionalLocales = Arrays.asList("en", "ru", "uk", "de", "pt", "pl", "ja", "es");
```

Comment:
@samartnik do we still need these regions ? i think it was added to ask users to upgrade for rewards.

Reply:
Yes, these locales to filter this specific notification. Not related to rewards.